lib/active_merchant/billing/gateways/pay_trace.rb

```diff
@@ -165,15 +162,35 @@ def acquire_access_token
 
       private
 
+      def build_purchase_request(money, payment_or_customer_id, options)
+        post = {}
+        add_amount(post, money, options)
+        if customer_id?(payment_or_customer_id)
+          post[:customer_id] = payment_or_customer_id
+        else
+          add_payment(post, payment_or_customer_id)
+          add_address(post, payment_or_customer_id, options)
+          add_customer_data(post, options)
+        end
+
+        post
+      end
+
+      def capture_request(money, authorization, options)
```

Comment:
this is a totally nitpicky thing to notice or mention but can we have either both or neither of these start with `build_`?

Reply:
Love it - yes and done!